core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -100,6 +101,10 @@
 map(Routing, Payload) ->
     map_processor(Routing, Payload, get_binding_candidates(Routing)).
 
+-spec map(ne_binary(), payload(), kz_bindings()) -> map_results().
```

Comment:
@jamesaimonetti that's part of a possible optimization for csv processing. you call candidates once and call map/3 for each row